tests/template_backends/test_django.py

```diff
@@ -176,27 +176,28 @@ def test_autoescape_default(self):
             "Hello, Bob &amp; Jim",
         )
 
-    default_loaders = [
-        "django.template.loaders.filesystem.Loader",
-        "django.template.loaders.app_directories.Loader",
-    ]
-
-    @override_settings(DEBUG=False)
-    def test_non_debug_default_template_loaders(self):
-        engine = DjangoTemplates(
-            {"DIRS": [], "APP_DIRS": True, "NAME": "django", "OPTIONS": {}}
-        )
-        self.assertEqual(
-            engine.engine.loaders,
-            [("django.template.loaders.cached.Loader", self.default_loaders)],
-        )
-
-    @override_settings(DEBUG=True)
-    def test_debug_default_template_loaders(self):
-        engine = DjangoTemplates(
-            {"DIRS": [], "APP_DIRS": True, "NAME": "django", "OPTIONS": {}}
-        )
-        self.assertEqual(engine.engine.loaders, self.default_loaders)
+    def test_default_template_loaders(self):
+        """
+        The cached template loader is alwasys enabled by default.
+        """
+        for debug in (True, False):
+            with self.subTest(DEBUG=debug):
+                with override_settings(DEBUG=debug):
```

Comment:
We can keep them together
```suggestion
            with self.subTest(DEBUG=debug), self.settings(DEBUG=debug):
```